src/main.ts

```diff
@@ -11,18 +11,91 @@ import { promisify } from 'util';
 import { exec } from 'child_process';
 import portfinder from 'portfinder';
 import * as TOML from '@iarna/toml';
+import axios from 'axios';
 
 
 import * as context from './context';
 import * as stateHelper from './state-helper';
+import { get } from 'http';
 
 const supportedDockerDriver = 'remote';
 const mountPoint = '/var/lib/buildkit';
 const device = '/dev/vdb';
+const mmdsIPv4Addr = "169.254.169.254";
```

Comment:
is this supposed to be hardcoded?

Reply:
yeah this is the FC specific IP where MMDS is shared